src/libcalamaresui/ViewManager.cpp

```diff
@@ -189,28 +192,106 @@ ViewManager::insertViewStep( int before, ViewStep* step )
 void
 ViewManager::onInstallationFailed( const QString& message, const QString& details )
 {
+bool    shouldOfferWebPaste = true; // TODO: config var
```

Comment:
Btw at least that TODO need be implemented.

Without that we force that on all the time.